internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -392,3 +393,67 @@ func (r *BundleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.ConfigMap{}).
 		Complete(r)
 }
+
+func (r *BundleReconciler) bundleImagePod(pod *corev1.Pod, source rukpakv1alpha1.ImageSource) *corev1.Pod {
+	if len(pod.Spec.InitContainers) != 1 {
+		pod.Spec.InitContainers = make([]corev1.Container, 1)
+	}
+
+	pod.Spec.InitContainers[0].Name = "install-unpacker"
+	pod.Spec.InitContainers[0].Image = r.UnpackImage
+	pod.Spec.InitContainers[0].ImagePullPolicy = corev1.PullIfNotPresent
+	pod.Spec.InitContainers[0].Command = []string{"cp", "-Rv", "/unpack", "/util/unpack"}
+	pod.Spec.InitContainers[0].VolumeMounts = []corev1.VolumeMount{{Name: "util", MountPath: "/util"}}
+
+	if len(pod.Spec.Containers) != 1 {
+		pod.Spec.Containers = make([]corev1.Container, 1)
+	}
+
+	pod.Spec.Containers[0].Name = bundleUnpackContainerName
+	pod.Spec.Containers[0].Image = source.Ref
+	pod.Spec.Containers[0].Command = []string{"/bin/unpack", "--bundle-dir", "/"}
+	pod.Spec.Containers[0].VolumeMounts = []corev1.VolumeMount{{Name: "util", MountPath: "/bin"}}
+
+	return pod
+}
+
+func (r *BundleReconciler) bundleGitRepoPod(pod *corev1.Pod, source rukpakv1alpha1.GitSource) (*corev1.Pod, error) {
+	// Note: use host networking (hack) so we can resolve https://github.com references in the
```

Comment:
nit: Should we make this a todo comment so an issue gets created?

Reply:
We ended up pulling this code out and the e2e still passed, so we are removing it altogether